administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -492,8 +494,11 @@ protected function downloadPackage($url, $target)
             return false;
         }
 
+        // Fix Indirect Modification of Overloaded Property
+        $body = $response->body;
```

Comment:
```suggestion
        $body = $result->body;
```
Here it is `$result`.